### DISCUSSION TITLE
Ani-cli not starting in windows.

### DISCUSSION THREAD

**Discussion by forsyth47** [pystardust/ani-cli#157] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88034696/144074115-4f0f6b02-1f7d-467d-a0f6-6e8bcc8de87b.png)

When ever i start try to start ani-cli, it tell me to open with some app
I tired opening it with shell/cmd but it dosnt start.

I jus donno what to do! T~T

I installed it in linux and it was so easy... it looks so complicated in windows...

**Comment by Hlaxfors** [comment]
I was equally confused.
It seems you can only run the command in the GitBash program you used to install it.

**Reply by ghost** [reply]
This. Shell scripting on Windows is not a native thing, It will only ever work on Powershell/CMD if someone makes a powershell-windows-specfic version of ani-cli.

**Comment by port19x** [comment, marked as answer]
As said in #185 support for platforms other than linux is less than ideal at the moment.
In the sadly somewhat distant future, we will probably release an exe.
Until then use GitBash
